internal/file/file_plugin.go

```diff
@@ -133,11 +133,28 @@ func (fp *FilePlugin) Subscriptions() []string {
 		bus.ConfigUploadRequestTopic,
 		bus.ConfigApplyRequestTopic,
 		bus.ConfigApplyFailedTopic,
-		bus.ConfigApplySuccessfulTopic,
+		bus.ReloadSuccessfulTopic,
 		bus.ConfigApplyCompleteTopic,
 	}
 }
 
+func (fp *FilePlugin) CleanUpConfigApply(ctx context.Context,
```

Comment:
```suggestion
func (fp *FilePlugin) enableFileWatcher(ctx context.Context,
```
Would it be better to move out the `fp.fileManagerService.ClearCache()` and just have this function enable the watcher?

So that it looks something like this:
```
fp.fileManagerService.ClearCache()
fp.enableFileWatcher(ctx, successMessage.ConfigContext, successMessage.DataPlaneResponse.GetInstanceId())
```